roles/ceph-common/tasks/installs/debian_ceph_repository.yml

```diff
@@ -38,21 +38,39 @@
   changed_when: false
   when: ceph_stable_uca
 
-- name: add custom repo
+- name: fetch nfs-ganesha development apt repo file
```

Comment:
@leseb would it make sense to move the nfs-ganesha repo generation and installation tasks to files under roles/ceph-nfs/tasks since they only apply to nfs nodes?